src/main/scala/com/codacy/clangtidy/ClangTidyResult.scala

```diff
@@ -0,0 +1,33 @@
+package com.codacy.clangtidy
+import java.nio.file.Path
+
+import com.codacy.model.Issue
+import com.codacy.model.Issue.LineLocation
+import com.codacy.plugins.api.results
+import com.codacy.plugins.api.results.Result
```

Comment:
You imported `results` and  `results.Result`

Reply:
Choose one

Reply:
Fixed